clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/api/KubernetesApiConverter.groovy

```diff
@@ -808,11 +808,19 @@ class KubernetesApiConverter {
         builder = builder.withNewRollingUpdate()
 
         if (rollingUpdate.maxSurge) {
-          builder = builder.withNewMaxSurge(rollingUpdate.maxSurge)
+          def maxSurge = rollingUpdate.maxSurge
```

Comment:
I'd prefer to pull these into separate methods, but the builder is overloaded to take an int or a string, and the builder changes type with each invocation depending on how nested you are in the build so you can't pass that to a helper.

I'll clean up the exception handler for clarity